feat(files): Add support for `storage-id` and `path` in RepairTree command

artonge opened the pull request:
This allows to run the command on specific part of the filecache. When the filecache is big, it prevents overloading the DB.

come-nc commented on apps/files/lib/Command/RepairTree.php:
```diff
@@ -26,11 +27,16 @@ protected function configure(): void {
 			->setDescription('Try and repair malformed filesystem tree structures (may be necessary to run multiple times for nested malformations)')
-			->addOption('dry-run');
+			->addOption('dry-run')
+			->addOption('storage-id', 's', InputOption::VALUE_OPTIONAL, 'If set, only repair files within the given storage ID', null)
```
```suggestion
			->addOption('storage-id', 's', InputOption::VALUE_OPTIONAL, 'If set, only repair files within the given storage numeric ID', null)
```
If I got it correctly this is the `numeric_id`?
At least it’s called numeric_id in DB and in the output of `occ info:storages`.
Are there other ways to know the storage ID?

artonge committed:
feat(files): Add support for `storage-id` and `path` in RepairTree command

This allows to run the command on specific part of the filecache. When the filecache is big, it prevents overloading the DB.

Signed-off-by: Louis Chmn <louis@chmn.me>

kesselb reviewed (approved):
Thank you 🙏
